R/progress.R

```diff
@@ -24,7 +24,13 @@ progress_bar <- function(type) {
   first <- TRUE
 
   show_progress <- function(down, up) {
-    if (type == "down") {
+    if (type == "both") {
+      total <- 1
```

Comment:
down[[1]] is defined by the `content-length` response header so it is unknown until the upload has completed (I think)